app/forms/new_phone_form.rb

```diff
@@ -20,18 +20,18 @@ class NewPhoneForm
 
   alias_method :setup_voice_preference?, :setup_voice_preference
 
-  def initialize(user, setup_voice_preference: false)
-    self.user = user
-    self.otp_delivery_preference = user.otp_delivery_preference
-    self.otp_make_default_number = false
-    self.setup_voice_preference = setup_voice_preference
+  def initialize(user:, setup_voice_preference: false)
+    @user = user
+    @otp_delivery_preference = user.otp_delivery_preference
+    @otp_make_default_number = false
+    @setup_voice_preference = setup_voice_preference
```

Comment:
if we're no longer using the writer methods, let's change the `attr_accessor` above to `attr_reader`?

Reply:
Yeah I'd be on-board with that. I'll need to check that the accessor methods weren't being used outside the class implementation.